assets/src/dashboard/app/views/myStories/content/index.js

```diff
@@ -80,7 +80,7 @@ function Content({
                   stories={stories}
                   tags={tags}
                   users={users}
-                  viewStyle={view.style}
+                  view={view}
```

Comment:
passing whole view object in now instead of just the style so we can easily reference pageSize